### PR TITLE
Fix useless opset_import in onnx

### PR DESCRIPTION
**Pull request by houseroad**
Fix the multiple opset_import introduced by the Caffe2==>ONNX converter.

### CODE REVIEW

**Review by yinghai — APPROVED** [review]
LGTM

**Review by bddppq — CHANGES_REQUESTED** [review]
`make_model` is the place that's supposed to add these model meta information, so we should change `make_model` to set default values for opset_id domain and version (instead of setting them in `caffe2_net_to_onnx_model`).

**Comment by houseroad** [comment]
I changed the code, now we use make_model to set the opset_import.

File: caffe2/python/onnx/tests/c2_ref_test.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def test_relu_node_inplace(self):
             name="test",
             inputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])],
             outputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])])
-        c2_rep = c2.prepare(make_model(graph_def))
+        c2_rep = c2.prepare(make_model(graph_def), producer_name='caffe2-ref-test')
```
Comment (bddppq) [inline caffe2/python/onnx/tests/c2_ref_test.py]: You are now passing `producer_name` to `prepare`, not `make_model` :-)

Comment (houseroad) [reply caffe2/python/onnx/tests/c2_ref_test.py]: oops... ;-)

**Review by bddppq — APPROVED** [review]

**Comment by bddppq** [comment]
@caffe2bot retest this please

**Pull request merged**